substra/sdk/rest_client.py

```diff
@@ -89,9 +89,16 @@ def set_config(self, config, profile_name='default'):
             raise exceptions.BadConfiguration('Your configuration is outdated, please update it.')
 
         username = config['auth']['username']
+        password = keyring.get_password(profile_name, username)
+
+        if password is None:
+            raise exceptions.KeyringException(
+                Exception("Fetching password error: Check your keyring installation")
```

Comment:
```suggestion
               "Fetching password error: Check your keyring installation"
```

Reply:
Is there a reason for creating an exception inside an exception?

Reply:
No, i'm just not familiar with the exceptions in python and thought it was the right way to do it